module/Api.js

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable  */
+
+import fetch from 'cross-fetch';
```

Comment:
- [ ] You're using cross-fetch but you don't have it listed as a dependency in your `package.json`. Please kindly add it as this may prevent the application from working.
<img width="763" alt="scr2" src="https://user-images.githubusercontent.com/52934128/184508076-898866bd-9863-4806-a134-3154428257a0.PNG">